### PR TITLE
Readme and Workflows

### PR DESCRIPTION
**Pull request by AHReccese**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +126,47 @@
...
 
 ## Usage
-
+### Simple Linear Model Preparation 
+```pycon
+>>> from sklearn import datasets
+>>> from pymilo.pymilo_obj import Export, Import
+>>> from sklearn.linear_model import LinearRegression
+>>> import os
+
```
Comment (sadrasabouri) [inline README.md:135]: I think it's better to remove this extra line.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

**Review by sadrasabouri — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
@AHReccese Everything seems good here 💯 
But. now that `README.md ` is ready we need to add `version_check.py` script and add it to CI.
Take a look at this [link](https://github.com/sepandhaghighi/pycm/blob/master/Otherfiles/version_check.py)
We can leave it for another PR or handle it here. @sadrasabouri

**Comment by sadrasabouri** [comment]
> @AHReccese Everything seems good here 100 But, now that `README.md ` is ready we need to add `version_check.py` script and add it to CI. Take a look at this [link](https://github.com/sepandhaghighi/pycm/blob/master/Otherfiles/version_check.py) We can leave it for another PR or handle it here. @sadrasabouri

I will handle it here ASAP.

**Comment by sadrasabouri** [comment]
I updated `test.yml` and add `version_chek.py` + `meta.yaml`.
Please review it again guys @sepandhaghighi @AHReccese

**Review by sepandhaghighi — APPROVED** [review]

**Pull request merged**